generator/generator_md.py

```diff
@@ -0,0 +1,96 @@
+import sys
```

Comment:
# Invalid PULL Commits

** Invalid Commit Message: GENERATOR-1022Added testgenerator.py **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']